driver/src/models/account_state.rs

```diff
@@ -1,32 +1,13 @@
-use ethcontract::{Address, H256, U256};
-use serde::{Deserialize, Serialize};
+use ethcontract::Address;
 use std::collections::HashMap;
 
-#[derive(Serialize, Default, Deserialize, Clone, Debug, PartialEq)]
-#[serde(rename_all = "camelCase")]
-pub struct AccountState {
-    pub state_hash: H256,
-    pub state_index: U256,
-    balances: HashMap<Address, HashMap<u16, u128>>, // UserId => (TokenId => balance)
-    pub num_tokens: u16,
-}
+// (UserId TokenId) => balance
+#[derive(Clone, Debug, Default, PartialEq)]
+pub struct AccountState(pub HashMap<(Address, u16), u128>);
```

Comment:
nit: Doesn't need to be `pub`.

Reply:
Even if it is not needed, it should be but it is used in the paginated orderbook reader.